dali/fuse/dafuse.cpp

```diff
@@ -1046,6 +1047,9 @@ int main(int argc, char *_argv[])
         ls->setMaxDetail(TopDetail);
         lf->beginLogging();
     }
+#else
+    updateContainerizedLogMsgHandler(getCategoryLogMsgFilter(MSGAUD_all, MSGCLS_all, TopDetail));
```

Comment:
dafuse is not in use and won't be ported to work in containers. So better left unaltered.